### DISCUSSION TITLE
Jalali Calendar

### DISCUSSION THREAD

**Discussion by atareversei** [mantinedev/mantine#5515] [Feature requests]
Hi, I know there have been some previous questions and discussions on this topic. I wanted to get some recent information whether supporting other calendars is planned or not.

The problem seems to arise from the `DayJS` library that doesn't support changing the calendar and changing the underlying library right now for `Mantine` seems hard work.

**Comment by rtivital** [comment]
No, it is not planned to add Jalali calendar support

**Reply by atareversei** [reply]
Is it possible for me to study your code base and come up with a proposal and then participate in changing the Mantine dates to work with other calendars as well?

Sorry if I sound funny😅, I haven't done any open source work before but I think Mantine is awesome and this addition will be a step forward towards making Mantine more useful for lots of international teams and projects.

**Reply by rtivital** [reply]
Sorry, I do not want to maintain this feature as a part of the library, so most likely it will never be added with or without community contributions. 

You can publish a separate dates package on npm that handles Jalali calendar, you can copy the entire `@mantine/dates` package make adjustments and publish it.

Also I've found https://www.npmjs.com/package/mantine-dates-jalali, maybe it can be a good idea to collaborate with the author of this library to not start from scratch
